### ISSUE TITLE
Custom dataset

### ISSUE THREAD

**Issue by lacie-life** [minghanz/trafcam_3d#8]
Thank you for your great work. I am very impressed with your work and want to apply your network to other datasets. 

Can you suggest some data processing workflow and instructions for training?

Thank you !!!

**Comment by Cypher2k2** [comment]
Hello, were you able to install and run the project ? i will really appreciate if you can guide us throw the necessarily steps to make it work

**Comment by lacie-life** [comment]
Yes, I installed and ran the project, but only for my custom dataset. 
For run this project, first you need intall bev package follow this [link](https://github.com/minghanz/bev), then create bev image and label base on this [dataloader](https://github.com/minghanz/yolov3/blob/58a8968fd7a46c538f5b533d4605c55a78f6da1d/utils/datasets.py) and data folder like [this](https://github.com/minghanz/yolov3/blob/58a8968fd7a46c538f5b533d4605c55a78f6da1d/dataset_format.txt) The training workflow is similar with yolov3.
More notes about training output and inference from author [here](https://github.com/minghanz/yolov3/blob/58a8968fd7a46c538f5b533d4605c55a78f6da1d/notes%20on%20data%20flow.txt)
